src/test/java/io/kestra/plugin/influxdb/FluxTriggerTest.java

```diff
@@ -0,0 +1,86 @@
+package io.kestra.plugin.influxdb;
+
+import io.kestra.core.junit.annotations.KestraTest;
+import io.kestra.core.models.executions.Execution;
+import io.kestra.core.queues.QueueFactoryInterface;
+import io.kestra.core.queues.QueueInterface;
+import io.kestra.core.repositories.LocalFlowRepositoryLoader;
+import io.kestra.core.runners.FlowListeners;
+import io.kestra.core.runners.Worker;
+import io.kestra.core.schedulers.AbstractScheduler;
+import io.kestra.core.utils.IdUtils;
+import io.kestra.core.utils.TestsUtils;
+import io.kestra.jdbc.runner.JdbcScheduler;
+import io.micronaut.context.ApplicationContext;
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Flux;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.is;
+
+@KestraTest
+public class FluxTriggerTest {
+
+    @Inject
+    private ApplicationContext applicationContext;
+
+    @Inject
+    @Named(QueueFactoryInterface.EXECUTION_NAMED)
+    private QueueInterface<Execution> executionQueue;
+
+    @Inject
+    private FlowListeners flowListenersService;
+
+    @Inject
+    private LocalFlowRepositoryLoader localFlowRepositoryLoader;
+
+    @Test
+    void run() throws Exception {
+        Execution execution = triggerFlow();
+
+        System.out.println("Execution");
+        System.out.println(execution.getTrigger().getVariables());
+
+        List<Map<String, Object>> rows = (List<Map<String, Object>>) execution.getTrigger().getVariables().get("rows");
+
+        assertThat(rows.size(), greaterThan(0));
+        assertThat(rows.getFirst().containsKey("_value"), is(true));
+        assertThat(rows.getFirst().containsKey("_field"), is(true));
+        assertThat(rows.getFirst().containsKey("_measurement"), is(true));
```

Comment:
Could we directly test we get the expected valued for the rows size and content?